client/state/happychat/middleware.js

```diff
@@ -18,6 +19,19 @@ export const requestTranscript = ( connection, { getState, dispatch } ) => {
 	);
 };
 
+const sendBrowserInfo = ( connection, siteUrl ) => {
+	const siteHelp = `Site I need help with: ${ siteUrl }\n`;
+	const screenRes = ( typeof screen !== 'undefined' ) && `Screen Resolution: ${ screen.width }x${ screen.height }\n`;
```

Comment:
I think the more precise semantics in this case should be `typeof screen === 'object'` here. Just like [we did in state/index.jsx](https://github.com/Automattic/wp-calypso/blob/master/client/state/index.js#L118). Otherwise, a `int` or a `string` is still valid here. Even though that is almost impossible to happen, I think it still benefits to be precise here 😃 

The same goes to the following two expressions, so I'm not going to repeat this there :)